src/__tests__/ProductListItem.spec.js

```diff
@@ -0,0 +1,18 @@
+import { render, screen, fireEvent } from "@testing-library/vue";
+import ProductListItem from "../components/product/ProductListItem.vue";
+
+describe("ProductListItem.vue", () => {
+  it("Edit button is visible", () => {
+    render(ProductListItem);
+    expect(screen.getByRole("edit")).toBeVisible();
```

Comment:
If I'm not mistaken the suggested way would be to `getByRole('button', { name: 'edit'} )` instead of creating custom roles for buttons. 🤔  
Or you could always just simply use `getByText('edit')` as it's not a strict requirement to opt for `role` query. 😉